addon/index.js

```diff
@@ -4,6 +4,22 @@ import { getValue } from '@glimmer/tracking/primitives/cache';
 export { modifier, Modifier } from './-private/modifiers';
 export { Resource } from './-private/resources';
 
+function normalizeArgs(args) {
```

Comment:
This function could be useful as public util. I actually have a use case for it.

Reply:
tell me more <3 :D

Reply:
Gonna mark this as resolved, because we talked on Discord :D